app/controllers/accounts_controller.rb

```diff
@@ -63,7 +62,7 @@ def update
   # DELETE /accounts/1 or /accounts/1.json
   def destroy
     # @account.destroy!
-    @account.update(:status_id => 3)
+    @account.update(:status_id => 4)
```

Comment:
status_id 4 doesnt exist (1: officer, 2: member, 3: inactive). Is this a new status which is required?